src/task2/validators/user.ts

```diff
@@ -0,0 +1,39 @@
+import * as Joi from '@hapi/joi';
+import {
+  createValidator,
+  ValidatedRequestSchema,
+  ContainerTypes,
+} from 'express-joi-validation';
+
+const validator = createValidator({
+  passError: true,
+});
+
+const loginValidatator = Joi.string().trim().pattern(/^\S*$/);
+const passwordValidatator = Joi.string().pattern(/[a-zA-Z0-9]/);
```

Comment:
this regex still allows us to put one-letter or one-digit password, unfortunately. 
However, I think this is not the central part of the task, not critical